draft-packages/select/package.json

```diff
@@ -35,13 +35,15 @@
     "@kaizen/component-library": "^9.3.0",
     "@kaizen/deprecated-component-library-helpers": "^2.1.1",
     "@kaizen/draft-form": "^3.2.10",
+    "@kaizen/draft-tag": "^1.7.17",
     "@types/classnames": "^2.2.10",
     "@types/react-select": "3.0.22",
     "classnames": "^2.2.6",
     "react-select": "^3.1.0"
   },
   "devDependencies": {
     "elm-storybook": "cultureamp/elm-storybook#0.3.0",
+    "@cultureamp/elm-storybook": "cultureamp/elm-storybook#0.2.1",
```

Comment:
This confuses me. Other packages use `"elm-storybook": "cultureamp/elm-storybook#0.3.0",`, is there a reason this one is different?

Reply:
@ActuallyACat that looks like an issue with a merge, i didn't put that there...
I think it can just be taken out